spec/models/report_spec.rb

```diff
@@ -13,12 +13,12 @@
 
     @valid_post_report = {
       :item_id => @bob_post.id,
-      :item_type => 'post',
+      :item_type => 'Post',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.